### PR TITLE
Suppress subscription button

### PR DESCRIPTION
**Pull request by nickpell**
implements (2) of http://manage.dimagi.com/default.asp?188533#BugEvent.1061522

@orangejenny 

best read commit by commit

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -842,6 +842,44 @@ def __init__(self, *args, **kwargs):
...
+        self.subscription = subscription
+        super(SuppressSubscriptionForm, self).__init__(*args, **kwargs)
+
+        self.helper = FormHelper()
+        self.helper.form_class = 'form-horizontal'
+
+        fields = [
+            crispy.Div(
+                crispy.HTML('Warning: this can only be undone by a developer.'),
+                    css_class='alert alert-error',
```
Comment (dimagimon) [inline corehq/apps/accounting/forms.py]: E131 continuation line unaligned for hanging indent

**Comment by orangejenny** [comment]
Should this have any kind of help, or will be it be obvious to the people who use this page what "suppress" will do?

**Comment by orangejenny** [comment]
just minor comments

**Comment by nickpell** [comment]
@orangejenny re: https://github.com/dimagi/commcare-hq/pull/9645#issuecomment-165594176

I think Ops should have enough context here

**Comment by nickpell** [comment]
:white_check_mark:

**Pull request merged**